### PR TITLE
fix failing pip on Debian 12

### PR DESCRIPTION
**Pull request by wfurt**
this is stop gap fix to get the build going. Seems to go agains python recommendation, we may look for better fix. 

contributes to #811

### CODE REVIEW

File: src/debian/12/helix/amd64/Dockerfile [hunk 1]
```diff
@@ -34,9 +34,9 @@ RUN apt-get update && \
 ENV LANG=en_US.utf8
 
 RUN ln -sf /usr/bin/python3 /usr/bin/python && \
-    python -m pip install virtualenv==20.17.1 && \
-    python -m pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
-    python -m pip install ./helix_scripts-*-py3-none-any.whl
+    python3 -m pip install virtualenv==20.17.1 --break-system-packages && \
+    python3 -m pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
+    python3 -m pip install ./helix_scripts-*-py3-none-any.whl --break-system-package
```
Comment (MattGal) [inline src/debian/12/helix/amd64/Dockerfile:39]: isn't this `--break-system-packages`

File: src/debian/12/helix/arm64v8/Dockerfile [hunk 2]
```diff
@@ -36,10 +36,10 @@ RUN apt-get update && \
 ENV LANG=en_US.utf8
 
 RUN ln -sf /usr/bin/python3 /usr/bin/python && \
-    python -m pip install virtualenv==20.17.1 && \
-    python -m pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
+    python3 -m pip install virtualenv==20.17.1 --break-system-package && \
```
Comment (MattGal) [inline src/debian/12/helix/arm64v8/Dockerfile:39]: Same for these two.

File: src/debian/12/helix/amd64/Dockerfile (hunk 1 above)
Comment (wfurt) [reply src/debian/12/helix/amd64/Dockerfile:39]: probably cut & paste error.
But it seems to work 
```
root@c76f85a8e096:/#  python3   -m pip install virtualenv==20.17.1 --break-system-package
Requirement already satisfied: virtualenv==20.17.1 in /usr/local/lib/python3.11/dist-packages (20.17.1)
Requirement already satisfied: distlib<1,>=0.3.6 in /usr/local/lib/python3.11/dist-packages (from virtualenv==20.17.1) (0.3.6)
Requirement already satisfied: filelock<4,>=3.4.1 in /usr/local/lib/python3.11/dist-packages (from virtualenv==20.17.1) (3.9.0)
Requirement already satisfied: platformdirs<3,>=2.4 in /usr/local/lib/python3.11/dist-packages (from virtualenv==20.17.1) (2.6.2)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
```

Comment (MattGal) [reply src/debian/12/helix/amd64/Dockerfile:39]: huh ok then

**Review by MattGal — APPROVED** [review]

File: src/debian/12/helix/amd64/Dockerfile (hunk 1 above)
Comment (wfurt) [reply src/debian/12/helix/amd64/Dockerfile]: updated anyway for consistency.

**Pull request merged**